mapeditor/mainwindow.cpp

```diff
@@ -429,6 +446,87 @@ void MainWindow::on_actionOpen_triggered()
 	openMap(filenameSelect);
 }
 
+void MainWindow::on_actionOpenRecent_triggered()
+{
+	QSettings s(Ui::teamName, Ui::appName);
+	QStringList recentFiles = s.value(recentlyOpenedFilesSetting).toStringList();
+
+	struct RecentFileDialog : public QDialog {
+		
+		RecentFileDialog(QWidget *parent, const QStringList& recentFiles) : QDialog(parent), layout(this) {
```

Comment:
parent traditionally goes last
```suggestion
		RecentFileDialog(QStringList recentFiles, QWidget *parent = nullptr) : QDialog(parent), layout(this)
		{
```

Reply:
Committed suggestion with [5bff1ec](https://github.com/vcmi/vcmi/pull/4939/commits/5bff1ec2c1b1ef045d92795152c0d088b1a1699f), accedentially undid it with [2678bf1](https://github.com/vcmi/vcmi/pull/4939/commits/2678bf1e05a89a694c39ca8f3f8f78ee3a5d260e) while rebasing, re-introduced with [9b0e676](https://github.com/vcmi/vcmi/pull/4939/commits/9b0e6763e76c54b817c0854d2b7eeec48a8bdf59).